### PR TITLE
VPN (via nmcli) 1.1 and API 0.5 compatiblity

### PR DESCRIPTION
**Pull request by Bierchermuesli**
i forked the vpn plugin in a dedicated branch - I think this would be ready to merge

### CODE REVIEW

**Comment by roman-acumen** [comment]
its the same as #154 right?

**Comment by ManuelSchneid3r** [comment]
Yes please and call these nm, network manager or something that makes it clear that nm is used. Otherwise it will work expectation when ppl use sth else. Imho the name should represent the scope. Vpn narrows the nm scope (whatever one could imagine manipulating interfaces printing net things) while not being complete in the sense that it does not interface other von providers.

File: vpn/__init__.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
+"""VPN Extension
+
```
Comment (ManuelSchneid3r) [inline vpn/__init__.py:2]: this text will apear in the setings dialog. please use it only if it is beneficial for users. title is in the config and will be duplicated.

File: vpn/__init__.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+Connect or disconnect from a network manager VPN profile"""
+
+
+from albert import *
+from collections import namedtuple
+import subprocess
+
+md_iid = "0.5"
+md_version = "0.5"
```
Comment (ManuelSchneid3r) [inline vpn/__init__.py:11]: __version__ = "0.4.0" was iidmajor.iidminor.version there fore 1.1 makes sene i guess

File: vpn/__init__.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+md_id = "vpn"
```
Comment (ManuelSchneid3r) [inline vpn/__init__.py:12]: you can drop this it will automatically be set to the dir name

File: vpn/__init__.py [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+md_name = "VPN"
+md_description = "Manages VPN (via nmcli)"
```
Comment (ManuelSchneid3r) [inline vpn/__init__.py:14]: Please from the user POV "Manage VPN…

File: vpn/__init__.py [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+md_license = "MIT"
+md_url = "https://github.com/albertlauncher/python"
+md_maintainers = ["@Bierchermuesli"]
+md_authors = ["@janeklb"]
```
Comment (ManuelSchneid3r) [inline vpn/__init__.py:18]: this field does not exist

File: vpn/__init__.py [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+md_lib_dependencies = ["nmcli"]
```
Comment (ManuelSchneid3r) [inline vpn/__init__.py:19]: is this really this one https://pypi.org/project/nmcli/ ? important otherwise dep mgmt will not work

Comment (ManuelSchneid3r) [reply vpn/__init__.py:19]: wait it does not even use this package. use `md_bin_dependencies`

**Review by ManuelSchneid3r — CHANGES_REQUESTED** [review]

**Pull request merged**